components/shared/AuthModal/components/login/LoginForm.tsx

```diff
@@ -55,6 +55,7 @@ export const LoginForm: FC<{ closeModal: TNoop; showRegisterForm: TNoop }> = ({
       const error = e as IResponseError;
       if (error.status === 404) {
         setAlert('Login yoki parol xato kiritilgan!');
+        setAlert('');
```

Comment:
you should clear form errors not the alert

Reply:
Yeh,ok but I cannot log in to test it